### ISSUE TITLE
Include the list of tagged repositories in the release notes

### ISSUE THREAD

**Issue by kdmccormick** [openedx/build-test-release-wg#241]
More info coming soon

**Comment by schenedx** [comment]
The list is at least generated now on [Olive release page on openedx confluence](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3604807768/Olive.1). It would be great in new release notes to call out this list, and potentially the net changes to this list.

**Comment by sambapete** [comment]
According to what I found while looking at the edx and openedx org, the list might be incomplete.

I found the following repos were also tagged with olive.1 but not mentionned on Confluence
edx/enterprise-access
edx/frontend-app-learner-dashboard
openedx/docs.openedx.org
openedx/openedx-demo-course
openedx/openedx-i18n
openedx/openedx-test-course

They may not be part of the release, but they were tagged. Should they be mentioned?

**Comment by arbrandes** [comment]
@sambapete, you're right, I missed some.  I had a mistake in the command I used to generate the list.  Here's [the diff](https://openedx.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=3604807768&selectedPageVersions=3&selectedPageVersions=2) of the updated version.

As we can see, though, openedx-i18n is not there.  This is because it doesn't have either an `openedx.yaml` file or an `open-release/olive.master` branch.  I have no clue why that is, if it's meant to be tagged.  Looks like @regisb did the tagging manually.

Régis, can you help us understand how that repo is supposed to work in terms of release tagging?

**Comment by regisb** [comment]
This repo contains the i18n strings after they have been uploaded to transifex. Thus, it should not be tagged at the same time as other repos. Whenever a new minor release is made, I re-download, validate and compile the i18n strings. I then push the compiled strings to master and tag the commit. This commit is checked out in the openedx Docker image that ships with tutor ([here](https://github.com/overhangio/tutor/blob/380786219e8e7af9c429eda8b5e47117d7af6b5e/tutor/templates/build/openedx/Dockerfile#L58)).

With the upcoming [openedx-translations](https://github.com/openedx/openedx-translations) repo (enacted by [OEP 58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html) I expect that the openedx-i18n repo will become obsolete. Thus I suggest that we do nothing about it for now :)

**Comment by arbrandes** [comment]
Great, thanks for clearing that up!

**Comment by kdmccormick** [comment]
I added this step to the release tagging process: https://openedx.atlassian.net/wiki/spaces/COMM/pages/19662426/Process+to+Create+an+Open+edX+Release#2e.-Add-the-list-of-tagged-repositories-to-the-notes-page
